bundler.plugins.d/foreman_ansible_core.rb

```diff
@@ -0,0 +1 @@
+gem 'foreman_ansible_core'
```

Comment:
This ought to be a requirement in the  gemspec too, I think, the plugin doesn't work without it